recipes/festival/meta.yaml

```diff
@@ -0,0 +1,97 @@
+{% set name = "festival" %}
+{% set version = "2.5.1" %}
+
+package:
+  name: {{ name|lower }}-split
+  version: {{ version }}
+
+source:
+  url: https://github.com/festvox/festival/archive/5db6ee949f4e4e9da25c87bd03b90e69f9011393.tar.gz
+  sha256: 9f0d0e4f0e91fca4430ee4a6ca4d3d24bac444f36b4f4e9653c6119aadfae13e
+
+build:
+  run_exports:
+    - {{ pin_subpackage('festival', max_pin='x.x') }}
+  skip: True  # [not linux]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make  # [linux]
+    - libgomp  # [linux]
+    - libestools-dev  # [linux]
+    - libestools-static  # [linux]  This is needed to satisfy Make, but not used
+
+  host:
+    - libestools  # [linux]
+  run:
+    - libestools  # [linux]
```

Comment:
Why are all these dependencies selected for linux if you are skipping `not linux`?

Reply:
This was probably some preparation for easier adding of other OS builds. But as that is improbably, I'm dropping these selectors.